content/en/docs/architecture/release-gating.md

```diff
@@ -4,8 +4,6 @@ date: 2020-10-28T11:14:39-04:00
 draft: false
 ---
 
-# Extending OpenShift Release Gates
```

Comment:
How is it a duplicate?

Reply:
![image](https://user-images.githubusercontent.com/30187067/97891863-526b4f00-1cfd-11eb-9b19-8657a053e61d.png)